src/services/preference/types.ts

```diff
@@ -10,6 +10,15 @@ export interface AudificationPreference extends Preference {
   readAfter: boolean;
 }
 
+export interface DataPreference extends Preference {
```

Comment:
I think you can make it `export type DataPreference = ModelSettings & Preference`?

Reply:
Actually `DataPreference` is not `DataCube`'s setting. It contains parameters you want users to try to create different datasets. I use a subset of `ModelSettings` just to demonstrate the playground works properly, and more meaningful preference items will be added in future PRs.